packages/next/client/image.tsx

```diff
@@ -360,6 +360,11 @@ export default function Image({
         ).join(',')}.`
       )
     }
+    if (layout !== 'fill' && (isNaN(widthInt!) || isNaN(heightInt!))) {
+      throw new Error(
+        `Image with src "${src}" has invalid "width" or "height" property. These should be numeric values.`
+      )
+    }
```

Comment:
```suggestion
    if (
      (typeof widthInt !== 'undefined' && isNaN(widthInt)) ||
      (typeof heightInt !== 'undefined' && isNaN(heightInt))
    ) {
      throw new Error(
        `Image with src "${src}" has invalid "width" or "height" property. These should be numeric values.`
      )
    }
```

Reply:
You are really good! Thanks for your help!